### PR TITLE
Aidan buff system

### PR DESCRIPTION
**Pull request by Aidan-AIE**
Basically the entire buff system. There are still some extra things I want to put on it but it is fully functional for now.

### CODE REVIEW

File: Assets/Scripts/Player/PlayerControls.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using TMPro;
+using Unity.VisualScripting;
```
Comment (SkeleKing042) [inline Assets/Scripts/Player/PlayerControls.cs:6]: Remove?

File: ProjectSettings/GraphicsSettings.asset
Comment (SkeleKing042) [inline ProjectSettings/GraphicsSettings.asset:1]: Was this file changed intentionally?

**Review by SkeleKing042 — CHANGES_REQUESTED** [review]
See comments

**Review by Aidan-AIE — COMMENTED** [review]
wowza

File: ProjectSettings/GraphicsSettings.asset
Comment (Aidan-AIE) [reply ProjectSettings/GraphicsSettings.asset:1]: dunzo

**Review by SkeleKing042 — APPROVED** [review]
👍

**Pull request merged**